tests/test_comments_from_post.py

```diff
@@ -0,0 +1,11 @@
+"""Tests for the 'comments' (from post) capability."""
+
+from gogettr import PublicClient
+
+client = PublicClient()
+
+
```

Comment:
Could we add a test for pulling comments from a post with 0 replies, and a test for pulling comments with a known number of replies?

I realize that might be flaky—if someone replies in the future, for example—but I do think it's a case we should cover?

Here's a post with no comments: `p2zkgu`
Here's a post with 2 comments: `p2zp65`

Reply:
I'll add this test case.

Reply:
I'd say we keep the test until it fails (because of comment increase).